core/store/migrations/0040_create_job_proposals.go

```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up40 = `
+CREATE TABLE job_proposals (
+    id BIGSERIAL PRIMARY KEY,
+	spec TEXT NOT NULL,
+	status VARCHAR (50) NOT NULL,
+	job_id uuid NOT NULL,
+	feeds_manager_id int NOT NULL,
```

Comment:
Should these be fks?

Reply:
`job_id` should not be a FK because the proposal exists before a job is created.

However feeds_manager_id should be. I'll make that change